components/link/README.md

```diff
@@ -91,3 +82,8 @@ Alternately, you can apply link styles to a native `<a>` element by importing th
 ```
 
 Add the `d2l-link-main` or `d2l-link-small` CSS classes to the `<a>` element to apply those styles.
+
+## Accessibility
+ - The `d2l-link` component follows the W3C's best practices for the [Link Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/link/)
+ - Use `aria-label` when the content included in the `Default` slot does not provide enough context
+ - When `target` is set to `_blank`, screen reader users are informed that the link will open up in a new tab
```

Comment:
We have a similar treatment for sighted users, displaying the little "opens in new window" icon, should we mention that too since it's equally required by WCAG? I think it's under [Understanding Guideline 3.2 (Predictable)](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/consistent-behavior.html) but not explicitly mentioned there ... though it's covered by one of the "techniques" here: https://www.w3.org/TR/WCAG20-TECHS/G201.html

Reply:
Yeah, I can include that as well!

Reply:
I've now made those changes, so let me know about the phrasing!